source/partial-deep.d.ts

```diff
@@ -54,6 +61,21 @@ const partialSettings: PartialDeep<Settings, {recurseIntoArrays: true}> = {
 };
 ```
 
+You can prevent undefined values in recurse arrays and tuples by passing `{recurseIntoArrays: true; allowUndefinedInArrays: false}` as the second type argument:
+
+```
+import type {PartialDeep} from 'type-fest';
+
+interface Settings {
+	languages: string[];
+}
+
+const partialSettings: PartialDeep<Settings, {recurseIntoArrays: true; allowUndefinedInArrays: false}> = {
+	languages: [undefined] 	// Error
+	languages: []			// OK
+};
+```
+
```

Comment:
Should should be moved to the docs for the option.

Reply:
I move the example in the option documentation but I am not sure this was what you expected.
Moreover, in vs-code, when I put my cursor over the `PartialDeep` type, I see the documentation and example, but I do not see what is inside the `PartialDeepOptions` type, and so I do not see my example anymore.
I have done it in a separate commit to be able to rollback it if needed.

Reply:
@PicoI2 We link the option to the type, so the option is reachable from the type and the type is reachable from the option.

Refer [this commit](https://github.com/sindresorhus/type-fest/pull/1019/commits/bb180f4faba7a907c90cb47df360ddfde1acc56a).